Source/Coop/Components/CoopGameComponents/CoopGameComponent.cs

```diff
@@ -598,6 +598,26 @@ void ProcessQuitting()
                 return;
             }
         }
+        
+        /// <summary>
+        /// Handles a player going MissingInAction by staying in the game too long.
+        /// </summary>
+        void ProcessMissingInAction()
+        {
+            var gameTime = Singleton<AbstractGame>.Instance?.GameTimer.EscapeDateTime;
```

Comment:
You are already inside of CoopGameComponent, there is no needed to lookup for a already available variable.
```python
var gameTime = LocalGameInstance.GameTimer.EscapeDateTime;
```